code-challenges.test.js

```diff
@@ -32,22 +42,100 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("evenlyDivisibleByThree", () => {
+    it("Return a string interpolation stating if given object is or is not divisble by 3, undefined, or is an error", () => {
+        expect(evenlyDivisibleByThree(15)).toEqual('15 is divisible by three')
+        expect(evenlyDivisibleByThree(0)).toEqual('undefined')
+        expect(evenlyDivisibleByThree(-7)).toEqual('-7 is not divisible by three')
+    })
+} )  
+
+//Jest output: 
+// FAIL  ./jest.test.js
+// evenlyDivisibleByThree
+//   ✕ Return a string interpolation stating if given object is or is not divisble by 3 (2 ms)
+
+// ● evenlyDivisibleByThree › Return a string interpolation stating if given object is or is not divisble by 3
+
+//   ReferenceError: evenlyDivisibleByThree is not defined
+
+
+
 // b) Create the function that makes the test pass.
 
+const evenlyDivisibleByThree = (object) => {
+    if(object === 0) {
+        return 'undefined' 
+    } else if(object !== 0) {
+        return `${object} is not evenly divisible by three`
+    } else if(object % 3 === 0 ) {
+        return `${object} is evenly divisible by three`
+    } else return "error"
+}
+
+console.log(evenlyDivisibleByThree(object1.number)) //output: "15 is divisible by three"
```

Comment:
This is a great start! Consider performing the logic in the function like the object destructuring.